nixos/modules/services/networking/clightning.nix

```diff
@@ -0,0 +1,108 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.clightning;
+
+  mkLightningNode = cfg: network:
+    let
+      configFile = pkgs.writeText "clightning-${network}.conf" cfg.config;
+    in
+    {
+      description = "clightning ${network} node";
+
+      # clightning requires bitcoin-cli to talk to either spruned or bitcoind
+      path = [ pkgs.altcoins.bitcoind ];
+
+      after = [ "network.target" ];
```

Comment:
There's no `network.target` for user units (`systemctl --user list-units -t target`). You can remove this dependency.